src/Frame/Frame.cc

```diff
@@ -1832,6 +1837,18 @@ void Frame::SaveFile(const std::string& root_folder, const CARTA::SaveFile& save
         return;
     }
 
+    if (change_rest_freq) {
+        casacore::CoordinateSystem coord_sys = image->coordinates();
+        casacore::String errorMsg("");
```

Comment:
Just a minor issue about the code style. `errorMsg` would be better as `error_msg`.

Reply:
fixed.